test/unit/test_set_active_layer_tool.py

```diff
@@ -57,9 +58,8 @@ def create_identify_result(
         )
         feature = QgsVectorLayerUtils.createFeature(layer, geometry, {})
         layer.dataProvider().addFeature(feature)
+        QgsProject.instance().addMapLayer(layer)
 
-        # using the actual QgsMapToolIdentify.IdentifyResult causes
-        # fatal exceptions, mock probably is sufficient for testing
         results.append(
```

Comment:
If the mock results are not needed anymore in tests, should this function return anything anymore? So shoud these line be removed and ```def create_identify_layers(...) -> None```

Reply:
I thought that that info might come in handy in some tests but since they aren't used right now, might as well remove those.